apps/web/src/app/(legal)/_common/_styles/legal.module.css

```
@@ -98,3 +98,35 @@
   list-style-type: disc;
 }
+
+.root :global(table) {
+  width: 100%;
+  border-collapse: collapse;
+  border-left: none;
+  border-right: none;
+  margin-block: 8px;
+}
+
+.root :global(table),
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)